integrations/ragas/src/haystack_integrations/components/evaluators/ragas/metrics.py

```diff
@@ -139,22 +119,29 @@ class MetricDescriptor:
         to set the input types of the evaluator component.
     :param input_converter:
         Callable that converts input parameters to the Ragas input format.
+    :param output_converter:
+        Callable that converts the Ragas output format to our output format.
+        Accepts a single output parameter and returns a list of results derived from it.
     :param init_parameters:
         Additional parameters that need to be passed to the metric class during initialization.
     """
 
     metric: RagasMetric
     backend: Type[Metric]
+    input_validator: Callable[[Any], None]
```

Comment:
Missing docstring. Also, it seems like this callable is not actually used to validate the input, rather the metric parameters?

Maybe `metric_params_validator` as a name instead?